client/new-stack-editor/lib/views/statusbar.coffee

```diff
@@ -0,0 +1,18 @@
+kd = require 'kd'
+JView = require 'app/jview'
+
+
+module.exports = class Statusbar extends JView
+
+
+  constructor: (options = {}, data) ->
+
+    options.cssClass = kd.utils.curry 'statusbar', options.cssClass
+    data ?= { row: 1, column: 1, title: '...' }
```

Comment:
this is `data` which will be set after so in `option` there is no requirement to have `title`